compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -420,34 +421,35 @@ private void defineConstructs(BLangPackage pkgNode, SymbolEnv pkgEnv) {
         // Treat constants and type definitions in the same manner, since constants can be used as
         // types. Also, there can be references between constant and type definitions in both ways.
         // Thus visit them according to the precedence.
-        List<BLangNode> typeAndClassDefs = new ArrayList<>();
-        pkgNode.constants.forEach(constant -> typeAndClassDefs.add(constant));
-        pkgNode.typeDefinitions.forEach(typDef -> typeAndClassDefs.add(typDef));
+        List<BLangNode> moduleDefs = new ArrayList<>();
+        pkgNode.constants.forEach(constant -> moduleDefs.add(constant));
+        pkgNode.typeDefinitions.forEach(typDef -> moduleDefs.add(typDef));
+        pkgNode.xmlnsList.forEach(xmlns -> moduleDefs.add(xmlns));
         List<BLangClassDefinition> classDefinitions = getClassDefinitions(pkgNode.topLevelNodes);
-        classDefinitions.forEach(classDefn -> typeAndClassDefs.add(classDefn));
+        classDefinitions.forEach(classDefn -> moduleDefs.add(classDefn));
```

Comment:
Can just do addAll instead of .forEach and adding one by one. Can get rid of the `classDefinitions` variable too.